airbyte-webapp-e2e-tests/cypress/integration/connectorBuilder.spec.ts

```diff
@@ -1,41 +1,113 @@
-import { goToConnectorBuilderPage, startFromScratch, testStream } from "pages/connectorBuilderPage";
+import { goToConnectorBuilderPage, startFromScratch, enterUrlPath, testStream } from "pages/connectorBuilderPage";
 import {
+  acceptSchemaWithMismatch,
+  assertMaxNumberOfPages,
+  assertMaxNumberOfSlices,
+  assertMaxNumberOfSlicesAndPages,
+  assertMultiPageReadItems,
+  assertHasNumberOfSlices,
   assertTestReadItems,
   assertTestReadAuthFailure,
+  assertSchema,
+  assertSchemaMismatch,
+  assertSource404Error,
+  cleanUp,
   configureAuth,
   configureGlobals,
-  configureStream,
   configurePagination,
-  assertMultiPageReadItems,
+  configureStream,
+  configureStreamSlicer,
+  invalidAuth,
 } from "commands/connectorBuilder";
+
 import { initialSetupCompleted } from "commands/workspaces";
 
 describe("Connector builder", () => {
   before(() => {
     initialSetupCompleted();
     goToConnectorBuilderPage();
     startFromScratch();
-  });
-
-  it("Configure basic connector", () => {
     configureGlobals();
     configureStream();
   });
 
-  it("Fail on missing auth", () => {
+  afterEach(() => {
+    cleanUp();
+  });
+
+  /*
+  This test assumes it runs before "Read - Without pagination or partition router" since auth will be configured at that
```

Comment:
This was already the case before and it felt costly to fix so I just added a comment